src/oc-client.js

```diff
@@ -284,13 +284,12 @@ var oc = oc || {};
         ]
       },
       headers: { Accept: 'application/vnd.oc.unrendered+json' },
-      contentType: 'text/plain',
       crossDomain: true,
       success: function(apiResponse) {
-        if (apiResponse.renderMode === 'rendered') {
+        if (apiResponse[0].response.renderMode === 'rendered') {
           return cb(MESSAGES_ERRORS_GETTING_DATA);
```

Comment:
I just realised, the oc server can give an error too. eg: I just had an error because of graphql connection.
I noticed I get a response with an error property, so we can look for that, then there is a details property with the error details.

Reply:
should we do that as a second PR?

Reply:
I guess one thing is, the error object should follow some sort of shape, sending a string may be a problem... can we instead send a `new Error(MESSAGE);`?